Submodels/Parameterizations.py

```
@@ -674,43 +918,112 @@ def CheckOptimizedResult(b_0: float,
             plt.figure("Airfoil Shape")
             plt.plot(upper_x, upper_y, label="Reconstructed Upper Surface")
             plt.plot(lower_x, lower_y, label="Reconstructed Lower Surface")
             plt.plot(self.reference_data[:self.idx_LE + 1, 0], self.reference_data[:self.idx_LE + 1, 1], "-.", color="g")
             plt.plot(self.reference_data[self.idx_LE:, 0], self.reference_data[self.idx_LE:, 1], "-.", color="g")
+            plt.xlabel("x/c [-]")
+            plt.ylabel("y/c [-]")
             plt.show()
 
             return
 
+        def GetBounds(x):
```

Ruff (0.8.2)

Missing return type annotation for private function `GetBounds`

(ANN202)